### PR TITLE
Df/fix import payslips

### PR DESCRIPTION
**Pull request by khoido**

### CODE REVIEW

File: lib/keypay/sync/payslip.rb [hunk 1]
```diff
@@ -6,8 +6,10 @@ class Payslip
       def self.pull_all(eh_context, kp_context)
         kp_payslips = get_keypay_attrs(kp_context)
         kp_payslips = kp_payslips.map{|_, attrs| to_eh_attrs(attrs)}
-        options = { 'payslips' => kp_payslips }
-        eh_service(eh_context).bulk(options)
+        kp_payslips.in_groups_of((ENV['PAYSLIPS_PER_REQUEST']|| 10).to_i, false).each do |grouped_payslips|
+          options = { 'payslips' => grouped_payslips }
```
Comment (thukim) [inline lib/keypay/sync/payslip.rb]: :+1: `in_groups_of`. btw, `each_slice` is a little bit faster, compared to `in_groups_of`

**Comment by thukim** [comment]
:+1:

**Comment by thukim** [comment]
```
In Gemfile:
    bundler (~> 1.10) ruby

  Current Bundler version:
    bundler (1.9.5)
```
:trollface:

File: lib/keypay/sync/payslip.rb (hunk 1 above)
Comment (khoido) [inline lib/keypay/sync/payslip.rb]: 👍

File: lib/keypay/sync/payslip.rb [hunk 2]
```diff
@@ -6,8 +6,10 @@ class Payslip
       def self.pull_all(eh_context, kp_context)
         kp_payslips = get_keypay_attrs(kp_context)
         kp_payslips = kp_payslips.map{|_, attrs| to_eh_attrs(attrs)}
-        options = { 'payslips' => kp_payslips }
-        eh_service(eh_context).bulk(options)
+        kp_payslips.each_slice((ENV['PAYSLIPS_PER_REQUEST']|| 10).to_i) do |grouped_payslips|
+          options = { 'payslips' => grouped_payslips }
+          eh_service(eh_context).bulk(options)
+        end
```
Comment (thukim) [inline lib/keypay/sync/payslip.rb]: :hamster:

**Pull request merged**